src/test/java/com/desiato/puresynth/models/UserControllerTest.java

```diff
@@ -43,6 +45,28 @@ public void createUser_ShouldReturnCreatedUser() throws Exception {
                 .andDo(print());
     }
 
+    @Test
+    public void deleteUser_ShouldDeleteUserAndReturnOk() throws Exception {
```

Comment:
It would be helpful to write the test in the form: [give-when-then](https://www.j-labs.pl/blog-technologiczny/given-when-then-pattern-in-unit-tests/). It helps to break down the pattern visually.

It helps especially, if you start having more complex rules. I skipped this comment last time, as the test was very simple and obvious.

Reply:
That's a really good way of structuring a test! thank you, I will follow this pattern